rabbitmq/definitions.json

```diff
@@ -0,0 +1 @@
+{"users": [{"password_hash": "BYipq3D94qlyiZVOAAYLVdN1v8H0BOrOpM9SH6ma5aB354FA", "tags": "administrator", "name": "guest", "hashing_algorithm": "rabbit_password_hashing_sha256"}], "parameters": [], "topic_permissions": [], "vhosts": [{"name": "/"}], "queues": [{"name": "cm-dead-letter-queue", "auto_delete": false, "vhost": "/", "arguments": {"x-dead-letter-exchange": "exchange", "x-message-ttl": 10000}, "type": "classic", "durable": true}, {"name": "hip-data-flow-request-queue", "auto_delete": false, "vhost": "/", "arguments": {"x-dead-letter-exchange": "cm-dead-letter-exchange"}, "type": "classic", "durable": true}, {"name": "hip-consent-notification-queue", "auto_delete": false, "vhost": "/", "arguments": {"x-dead-letter-exchange": "cm-dead-letter-exchange"}, "type": "classic", "durable": true}, {"name": "hiu-consent-notification-queue", "auto_delete": false, "vhost": "/", "arguments": {"x-dead-letter-exchange": "cm-dead-letter-exchange"}, "type": "classic", "durable": true}, {"name": "consent-request-queue", "auto_delete": false, "vhost": "/", "arguments": {"x-dead-letter-exchange": "cm-dead-letter-exchange"}, "type": "classic", "durable": true}], "rabbitmq_version": "3.8.3", "policies": [], "exchanges": [{"name": "exchange", "durable": true, "vhost": "/", "internal": false, "arguments": {}, "type": "direct", "auto_delete": false}, {"name": "cm-dead-letter-exchange", "durable": true, "vhost": "/", "internal": false, "arguments": {}, "type": "topic", "auto_delete": false}, {"name": "hiservice.exchange.dataflowrequest", "durable": true, "vhost": "/", "internal": false, "arguments": {}, "type": "topic", "auto_delete": false}], "bindings": [{"vhost": "/", "destination": "cm-dead-letter-queue", "routing_key": "#", "source": "cm-dead-letter-exchange", "arguments": {}, "destination_type": "queue"}, {"vhost": "/", "destination": "hip-data-flow-request-queue", "routing_key": "hip-data-flow-request-queue", "source": "exchange", "arguments": {}, "destination_type": "queue"}, {"vhost": "/", "destination": "hip-consent-notification-queue", "routing_key": "hip-consent-notification-queue", "source": "exchange", "arguments": {}, "destination_type": "queue"}, {"vhost": "/", "destination": "hiu-consent-notification-queue", "routing_key": "hiu-consent-notification-queue", "source": "exchange", "arguments": {}, "destination_type": "queue"}, {"vhost": "/", "destination": "consent-request-queue", "routing_key": "consent-request-queue", "source": "exchange", "arguments": {}, "destination_type": "queue"}], "rabbit_version": "3.8.3", "permissions": [{"write": ".*", "vhost": "/", "read": ".*", "user": "guest", "configure": ".*"}], "global_parameters": [{"name": "cluster_name", "value": "rabbit@rabbitmq"}, {"name": "internal_cluster_id", "value": "rabbitmq-cluster-id-TmPfev8PqMAODZPsBGdQSg"}]}
```

Comment:
what is this?

Reply:
This user config, I have used the same which I exported from rabbitmq in the production environment(aws)